src/ExponentialMap.jl

```diff
@@ -86,140 +85,182 @@ function get_rows(spmexp::SparseMatrixExp,
 end
 
 """
-    ExponentialMap <: LazySet
+    ExponentialMap{S<:LazySet} <: LazySet
 
-Type that represents the action of an exponential map on a set.
+Type that represents the action of an exponential map on a convex set.
 
 ### Fields
 
-- `spmexp` -- a matrix exponential
-- `X`      -- a convex set represented by its support function
+- `spmexp` -- sparse matrix exponential
+- `X`      -- convex set
 """
 struct ExponentialMap{S<:LazySet} <: LazySet
     spmexp::SparseMatrixExp
     X::S
 end
-ExponentialMap(spmexp, X::S) where {S<:LazySet} = ExponentialMap{S}(spmexp,X)
 
-# instantiate an exponential map from a sparse matrix exponential
+"""
+    *(spmexp::SparseMatrixExp, X::LazySet)
+
+Return the exponential map of a convex set from a sparse matrix exponential.
+
+### Input
+
+- `spmexp` -- sparse matrix exponential
+- `X`      -- convex set
+
+### Output
+
+The exponential map of the convex set.
+"""
 function *(spmexp::SparseMatrixExp, X::LazySet)
     return ExponentialMap(spmexp, X)
 end
 
 """
-    dim(em)
+    dim(em::ExponentialMap)
 
-The ambient dimension of a ExponentialMap.
+Return the dimension of an exponential map.
 
 ### Input
 
 - `em` -- an ExponentialMap
+
+### Output
+
+The ambient dimension of the exponential map.
 """
-function dim(em::ExponentialMap)::Int64
+function dim(em::ExponentialMap)
     return size(em.spmexp.M, 1)
 end
 
-# support vector of the exponential map
-function σ(d::AbstractVector{Float64}, em::ExponentialMap)::AbstractVector{Float64}
-    v = expmv(1.0, em.spmexp.M.', d)          # v   <- exp(A') * d
-    res = expmv(1.0, em.spmexp.M, σ(v, em.X)) # res <- exp(A) * support_vector(v, S)
-    return res
+"""
+    σ(d::AbstractVector{Float64}, em::ExponentialMap)::AbstractVector{Float64}
+
+Return the support vector of the exponential map.
+
+### Input
+
+- `d`  -- direction
+- `em` -- exponential map
+
+### Output
+
+The support vector in the given direction.
+If the direction has norm zero, the result depends on the wrapped set.
+
+### Notes
+
+If ``E = \\exp(M)⋅S``, where ``M`` is a matrix and ``S`` is a convex set, it
+follows that ``σ(d, E) = \\exp(M)⋅σ(\\exp(M)^T d, S)`` for any direction ``d``.
+"""
+function σ(d::AbstractVector{Float64},
```

Comment:
also in this file. there seems to be an extra """ somewhere (code highlight is gone)